### PR TITLE
update to reflect new profiles api

### PR DESCRIPTION
**Pull request by ashley-joost**
- make daily request to new endpoint rather than scraping website
- update data in react components to reflect new structure

### CODE REVIEW

**Comment by ashley-joost** [comment]
@kaw2k i assigned you since you were the last to touch this :)

File: server.js [hunk 1]
```diff
@@ -36,68 +36,17 @@ setInterval(requestPeople, 1000*60*60*24);
...
-        });
-
-        //var random = _(people).shuffle().first(5).value();
-        //var selected = random[0];
-        //random = _.shuffle(random);
-
-        //fn({
-            //selected: selected,
-            //pool: people
-        //});
```
Comment (kaw2k) [inline server.js]: 👋

**Review by kaw2k — APPROVED** [review]
Sorry for delay, buried in a pile of stuff. Looks good, real api's are real 😂

**Comment by ashley-joost** [comment]
does this deploy automatically?

**Comment by kaw2k** [comment]
That is an excellent question. I know it is heroku, not sure if it auto deploys.

**Pull request merged**